[2024-05-24T03:27:30.473+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:27:29.082376+00:00 [queued]>
[2024-05-24T03:27:30.494+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:27:29.082376+00:00 [queued]>
[2024-05-24T03:27:30.495+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-05-24T03:27:30.515+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): execute_teste_py> on 2024-05-24 03:27:29.082376+00:00
[2024-05-24T03:27:30.524+0000] {standard_task_runner.py:57} INFO - Started process 1633 to run task
[2024-05-24T03:27:30.531+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'minio_python_task', 'execute_teste_py', 'manual__2024-05-24T03:27:29.082376+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag_teste/teste_***.py', '--cfg-path', '/tmp/tmp1mni5u7s']
[2024-05-24T03:27:30.533+0000] {standard_task_runner.py:85} INFO - Job 45: Subtask execute_teste_py
[2024-05-24T03:27:30.622+0000] {task_command.py:415} INFO - Running <TaskInstance: minio_python_task.execute_teste_py manual__2024-05-24T03:27:29.082376+00:00 [running]> on host 7b5dece41aaf
[2024-05-24T03:27:30.746+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_python_task' AIRFLOW_CTX_TASK_ID='execute_teste_py' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T03:27:29.082376+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-24T03:27:29.082376+00:00'
[2024-05-24T03:27:30.749+0000] {logging_mixin.py:151} INFO - Iniciando instalação do PySpark...
[2024-05-24T03:27:30.749+0000] {logging_mixin.py:151} INFO - Iniciando a instalação do Java...
[2024-05-24T03:27:30.775+0000] {logging_mixin.py:151} INFO - Erro durante a instalação do Java: Command '['apt-get', 'install', '-y', 'default-jre']' returned non-zero exit status 100.
[2024-05-24T03:27:30.775+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-24T03:27:30.786+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=minio_python_task, task_id=execute_teste_py, execution_date=20240524T032729, start_date=20240524T032730, end_date=20240524T032730
[2024-05-24T03:27:30.832+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-24T03:27:30.867+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
